WORKSPACE

```diff
@@ -5,3 +5,6 @@ load("//bazel:cc_configure.bzl", "cc_configure")
 
 envoy_dependencies()
 cc_configure()
+
+load("//bazel:git_version.bzl", "git_version")
```

Comment:
NIt: nominal Bazel style is to have all `load` at top of file (https://bazel.build/versions/master/docs/skylark/build-style.html), but it's not a big issue in a short WORKSPACE file to me, just a FYI.